ROADMAP.md

```diff
@@ -19,16 +40,13 @@ Reference: [Control plane issues](https://github.com/kserve/kserve/issues?q=is%3
 
 ## Objective: "Graduate KServe Python SDK to 1.0“
 
-- Improve KServe Python SDK dependency management with Poetry
 - Create standarized model packaging API
 - Improve KServe model server observability with metrics and distruted tracing
 - Support batch inference
 
 Reference：[Python SDK issues](https://github.com/kserve/kserve/issues?q=is%3Aissue+is%3Aopen+label%3Akserve%2Fsdk), [Storage issues](https://github.com/kserve/kserve/issues?q=is%3Aissue+is%3Aopen+label%3Akfserving%2Fstorage)
 
 ## Objective: "Graduate ModelMesh to beta"
```

Comment:
@ckadner Anything to update for ModelMesh?

Reply:
cc @israel-hdez @spolti